src/main/java/com/bootcamp/demo/dto/request/SignupRequest.java

```diff
@@ -0,0 +1,63 @@
+package com.bootcamp.demo.dto.request;
+
+import com.bootcamp.demo.dto.AbstractDTO;
+
+
+public class SignupRequest extends AbstractDTO {
+
+    private String firstname;
```

Comment:
To be consistent, you could rename this class to RegisterRequest, to avoid having multiple terms expressing the same concept (register, sign up, etc.). Pick one (that you like most, and stick with it, to track easier the flow).

You could use directly a model object instead of redefining the same attributes in various classes. By mistake, you could miss or add extra attributes to the user. In this case for example, you could keep it simple and have just for example `private User user` and in the constructor to instantiate that user.